jukebox_radio/streams/migrations/0003_auto_20210115_0506.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.1.2 on 2021-01-15 05:06
```

Comment:
@op1346 migration files you may ignore for now. They are auto generated. Typically you will not have to touch these.